snippets/python.json

```diff
@@ -214,6 +214,17 @@
         ],
         "description": "Add a method compute"
     },
+    "odoo_method_constrains": {
+        "prefix": "omconst",
+        "body": [
+            "@api.constrains('${1:field_name}')",
+            "def _check_${1:field_name}(self):",
+            "\if 1==1:",
```

Comment:
@elmerdpadilla thanks to PR! Great work! :+1: 

Please, fix this line. Change `\` by `\t`

```json
"\tif True:",
```